### PR TITLE
Dev/dbaas/codegen

### PR DESCRIPTION
**Pull request by bziouechn**

### CODE REVIEW

File: generators/service-hypersecure-dbaas-mongodb/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use strict'
+const BaseGenerator = require('../lib/generatorbase');
+const SCAFFOLDER_PROJECT_PROPERTY_NAME = "hypersecuredb";
```
Comment (gconan) [inline generators/service-hypersecure-dbaas-mongodb/index.js]: scaffolder property = `hypersecuredb`

File: generators/service-hypersecure-dbaas-mongodb/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const SERVICE_NAME = "service-hypersecure-dbaas-mongodb";
```
Comment (gconan) [inline generators/service-hypersecure-dbaas-mongodb/index.js]: service_name = `hypersecuredbaas`

File: generators/service-hypersecure-dbaas-mongodb/index.js (hunk 1 above)
Comment (bziouechn) [reply generators/service-hypersecure-dbaas-mongodb/index.js]: @gconan  , it is already set to hypersecuredb 
SCAFFOLDER_PROJECT_PROPERTY_NAME = "hypersecuredb"; 

Is there another change that i am not seeing ?

**Comment by gconan** [comment]
@Aerex @christiancompton Correct me if im wrong, but does the folder structure need to be `/generators/service-hypersecuredb` or is it ok the way it is?

**Comment by gconan** [comment]
nope, i looked at code, directory does not matter

**Pull request merged**